### PR TITLE
[1LP][RFR] Fix test_bad_password

### PR DESCRIPTION
**Pull request by apagac**
This is a fix for an existing test. Couple of things changed:

 - removed support for 5.7
 - added a BZ blocker for Chrome browser
 - using VersionPicker now
 - added support for 5.10

Not sure if I should leave the blocker there, since it only concerns Chrome browser and in Firefox this will work just fine.

{{ pytest: -v --long-running cfme/tests/test_login.py -k test_bad_password --use-provider vsphere67-nested }}

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit f630044382235741b9e92bd27131ad42d00522df:

`cfme/tests/test_login.py`:
- :bangbang: Line 57:9: **E123** *closing bracket does not match indentation of opening bracket's line*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

File: cfme/tests/test_login.py [hunk 1]
```diff
@@ -6,6 +6,8 @@
 from cfme.base.ui import LoginPage
 from cfme.utils import conf, version
```
Comment (sshveta) [inline cfme/tests/test_login.py]: Remove version from import
from cfme.utils import conf

File: cfme/tests/test_login.py [hunk 2]
```diff
@@ -39,17 +41,20 @@ def test_login(context, method, appliance):
...
     user = appliance.collections.users.instantiate(credential=cred, name='Administrator')
-    if appliance.version.is_in_series('5.7'):
-        error_message = "Sorry, the username or password you entered is incorrect."
-    else:
-        error_message = "Incorrect username or password"
+
+    error_message = VersionPicker({
+        version.LOWEST: "Incorrect username or password",
+        '5.10': "Login failed: Unauthorized"
+    }).pick(appliance.version)
```
Comment (sshveta) [inline cfme/tests/test_login.py]: error_message = VersionPicker({
        LOWEST: "Incorrect username or password",
        '5.10': "Login failed: Unauthorized"
    }).pick(appliance.version)

File: cfme/tests/test_login.py [hunk 3]
```diff
@@ -6,6 +6,8 @@
 from cfme.base.ui import LoginPage
 from cfme.utils import conf, version
 from cfme.utils.appliance import ViaSSUI, ViaUI
+from cfme.utils.blockers import BZ
+from cfme.utils.version import VersionPicker
```
Comment (sshveta) [inline cfme/tests/test_login.py]: from cfme.utils.version import VersionPicker, LOWEST

File: cfme/tests/test_login.py (hunk 2 above)
Comment (izapolsk) [reply cfme/tests/test_login.py]: OPTIONAL: I'd just use if appliance.version > 5.10. This construction is too sophisticated.

**Pull request merged**